gitgud/operations.py

```diff
@@ -215,3 +219,23 @@ def get_operator():
         if os.path.isdir(gg_path):
             return Operator(path)
     return None
+
+
+def init_tracking_json(file_operator):
+    print("Creating: Commit Tracker")
+    with open(os.path.join(file_operator.git_path, "gud", "commits.json"), 'w') as fp:
+        json.dump({}, fp)
+        print("Successfully created")
+        print()
```

Comment:
Remove all logging, this is an internal function, so seeing this isn't useful to the user.

Reply:
Logging removed